### ISSUE TITLE
404 errors on page

### ISSUE THREAD

**Issue by dcbroad3** [factoriolab/factorio-lab#248]
The website throws a 404 error on load, noted in the console and more noticeable on browser like Brave.

This is currently by design, the calculator redirects to the url "/list" or "/flow", and those pages do not exist. So, GitHub pages returns the 404.html page which we have replaced with the actual website itself. This is done so that angular routing can work. 

Theoretically, this could be made nicer by adding "list.html" and "flow.html" pages that are also copies of the "index.html" file, and those would then no longer return a 404 error.

**Comment by tzwaan** [comment]
why not do `https://factoriolab.github.io/#page=list&p=follower-robot-count*60` instead of `https://factoriolab.github.io/list#p=follower-robot-count*60`

**Comment by dcbroad3** [comment]
I think this would have to be more like:
https://factoriolab.github.io?p=follower-robot-count*60#list

If I were to switch to hashed routing in Angular, I wouldn't be able to also store the application state (the "p=...") inside the hash, it would have to be moved to query parameters or something else. 

Note that the hashed routing strategy also wouldn't address the question that was brought up in Discord with regards to doing something like `#electronic-circuit# to target that element in the page. In order to that it'd have to use the pattern:
https://factoriolab.github.io/list?p=follower-robot-count*60#electronic-circuit

In other words, that would require three separate parts: URL for routing, query parameter for application state, and hash for the target element in the page.
